boards/nrf52840dongle/Makefile.include

```diff
@@ -0,0 +1,28 @@
+# This board uses the vendor's serial bootloader
+
+PROGRAMMER ?= nrfutil
+
+ifeq (nrfutil,$(PROGRAMMER))
+
+# Using nrfutil implies using the Nordic bootloader described at
+# <https://devzone.nordicsemi.com/nordic/short-range-guides/b/getting-started/posts/nrf52840-dongle-programming-tutorial>.
+#
+# It has a static MBR at the first 4k, and an ample 128k Open USB Bootloader at
+# the end, leaving 892k for the application. This overwrites any SoftDevice,
+# but that's what the minimal working example for the dongle does as well.
+
+ROM_OFFSET = 0x1000
+FW_ROM_LEN = 0xdf000
+
+FLASHFILE = $(HEXFILE)
+FLASHDEPS += $(HEXFILE).zip
+FLASHER = nrfutil
+FFLAGS = dfu usb-serial --port=${PORT} --package=$(HEXFILE).zip
+
+%.hex.zip: %.hex
```

Comment:
This target could be moved outside the ifeq block.

Reply:
Thanks, fixed. (I was a bit unsure how to do this properly anyway with indentation).